hypertrace-graphql-span-processing-schema/src/main/java/org/hypertrace/graphql/spanprocessing/schema/rule/filter/SpanProcessingRelationalFilter.java

```diff
@@ -16,19 +17,27 @@ public interface SpanProcessingRelationalFilter {
 
   @GraphQLField
   @GraphQLName(SPAN_PROCESSING_FILTER_KEY_KEY)
+  @GraphQLDescription(
+      "Span processing filter key is the span attribute provided as string like http.request.body.email")
```

Comment:
nit - let's use well known attributes here for examples (e.g. "http.request.header.authorization")